README.md

```diff
@@ -28,3 +28,153 @@ Tentative roadmap:
 
 - VPN API
   - [ ] VPN API for desktop (Linux, Windows, macOS)
+
+# Cross-platform Development
+
+## Building
+
+In Go, you can compile for other target operating system and architecture by specifying the `GOOS` and `GOARCH` environment variables.
+
+<details>
+  <summary>Examples</summary>
+
+MacOS example:
+```
+% GOOS=darwin go build -C x -o ./bin/ ./outline-connectivity 
```

Comment:
*nit* since this is the SDK's README, are we OK to use `outline-connectivity` as sample? Also should the path be `go build ./x/outline-connectivity` if you are executing this command under the SDK root folder?

Reply:
That's the only binary we have.

We can't use `go build ./x/outline-connectivity` because it's not in the root module.